.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -0,0 +1,120 @@
+name: rawls-run-azure-e2e-tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch of rawls to run tests on'
+        required: true
+        default: 'develop'
+        type: string
+
+env:
+  BEE_NAME: '${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt}}-dev'
+  TOKEN: '${{ secrets.BROADBOT_TOKEN }}' # github token for access to kick off a job in the private repo
+
+jobs:
+  rawls-build-tag-publish-job:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    outputs:
+      custom-version-json: ${{ steps.render-rawls-version.outputs.custom-version-json }}
+    steps:
+      - uses: 'actions/checkout@v3'
+
+      - name: Bump the tag to a new version
+        uses: databiosphere/github-actions/actions/bumper@bumper-0.2.0
+        id: tag
+        env:
+          DEFAULT_BUMP: patch
+          GITHUB_TOKEN: ${{ env.TOKEN }}
+          RELEASE_BRANCHES: main
```

Comment:
Does this need to be `develop` as that's the "main" branch for Rawls?

Reply:
tbh, I don't know how how this action works and what RELEASE_BRANCHES refers to. But I can assure you that it's creating the release of Rawls from the commit that is at the head of the branch being supplied as as argument (I have looked through the logs and seen it check out the right commit, and the release that is built and pushed up uses the commit SHA as part of its name).

Looking at https://github.com/DataBiosphere/github-actions/blob/master/actions/bumper/action.yml doesn't help. But I took this from the existing swat workflow, so I am hoping this argument is correct.

Reply:
It looks like "main" is probably ignored here.

```
Is hotfix.* a match for WOR-916
hotfix_release = false
Is main a match for WOR-916
pre_release = true
From https://github.com/broadinstitute/rawls

Reply:
From the README in that link
> RELEASE_BRANCHES (optional) - Comma separated list of branches (bash reg exp accepted) that will generate the release tags. Other branches and pull-requests generate versions postfixed with the commit hash and do not generate any tag. Examples: master or .* or release.*,hotfix.*,master ...

I'm not that familiar with the release pipeline these days, but it does seem like a non-issue